includes/html/print-graylog.inc.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2014 Neil Lathwood <https://github.com/laf/ http://www.lathwood.co.uk/fa>
```

Comment:
Just inline this file in the overview/graylog.inc.php.

Also, you should not be assigning copyright to other people.  Please fix in all files.

Reply:
Ok, this was copy and paste as I copied print-syslog.inc.php and modified it for Graylog.
Should I add Copyright to overview/graylog.inc.php?
Every other file existed before, because of that I did not change / add any copyright, or should I change anything here?

Reply:
You may add your copyright on the files, that is up to you.  All files will be licensed under GPLv3 as per our license.  Individual file license headers are just for good measure.